backend/vaa-strapi/src/api/answer/routes/answer.ts

```diff
@@ -3,7 +3,7 @@
  */
 
 import { factories } from '@strapi/strapi';
-import { restrictResourceOwnedByCandidate, restrictPopulate, restrictFilters, electionCanEditQuestions } from '../../../util/acl';
```

Comment:
This is just renaming the property more consistently (and reversing the boolean logic). It effects a gazillion files, tho 😅